src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -841,6 +886,37 @@ resource trigger_DailySchedule 'Microsoft.DataFactory/factories/triggers@2018-06
   }
 }
 
+resource trigger_RecommendationsDailySchedule 'Microsoft.DataFactory/factories/triggers@2018-06-01' = {
```

Comment:
Any reason we need to create separate schedule? We already have one for exports. Can we just use one schedule so everything runs at the same time?

Reply:
I don't see a problem in using the same schedule. The only problem is customers who just deployed hubs will have to wait until the next day to see recommendations. I decided for a specific schedule so that we set the trigger start time to be 1h after the deployment time.